src/ViewFactory.cs

```diff
@@ -57,28 +86,42 @@ public static class ViewFactory
+                                              || viewDescendantType == typeof( Window ) );
+
+    private static bool IsSupportedType( this Type t )
+    {
+        return t == typeof( Window ) || ( !KnownUnsupportedTypes.Any( t.IsSubclassOf ) & !KnownUnsupportedTypes.Contains( t ) );
+    }
 
     /// <summary>
-    /// Creates a new instance of a <see cref="View"/> of Type <typeparamref name="T"/> with
-    /// size/placeholder values that make it easy to see and design in the editor.
+    ///   Creates a new instance of a <see cref="View" /> of Type <typeparamref name="T" /> with
+    ///   size/placeholder values that make it easy to see and design in the editor.
     /// </summary>
-    /// <typeparam name="T">A descendant of <see cref="View"/> that does not exist in the
-    /// <see cref="KnownUnsupportedTypes"/> collection.</typeparam>
-    /// <param name="width">The width of the requested view.</param>
-    /// <param name="height">The height of the requested view.</param>
-    /// <param name="text">Optional string to set the Text property of types that have one.</param>
-    /// <exception cref="NotSupportedException">If an unsupported type is requested.</exception>
-    /// <returns>A new instance of <paramref name="{T}"/>.</returns>
+    /// <typeparam name="T">
+    ///   A concrete descendant type of <see cref="View" /> that does not exist in the
+    ///   <see cref="KnownUnsupportedTypes" /> collection and which has a public constructor.
+    /// </typeparam>
+    /// <param name="width">
+    ///   An optional width of the requested view. Default values are dependent on the requested
+    ///   type, if not supplied.
+    /// </param>
+    /// <param name="height">
+    ///   An optional height of the requested view. Default values are dependent on the requested
+    ///   type, if not supplied.
+    /// </param>
+    /// <exception cref="NotSupportedException">If an unsupported type is requested</exception>
+    /// <returns>
+    ///   A new instance of <paramref name="{T}" /> with the specified dimensions or defaults, if not provided.
+    /// </returns>
     /// <remarks>
-    /// <typeparamref name="T"/> must inherit from <see cref="View"/>,
-    /// must have a public constructor, and must not exist in the
-    /// <see cref="KnownUnsupportedTypes"/> collection, at run-time.
+    ///   <typeparamref name="T" /> must inherit from <see cref="View" />, must have a public constructor, and must
+    ///   not exist in the <see cref="KnownUnsupportedTypes" /> collection, at run-time.
     /// </remarks>
     public static T Create<T>(int? width = null, int? height = null, string? text = null )
         where T : View, new( )
     {
-        if ( !SupportedViewTypes.Contains( typeof( T ) ) )
+        if ( !IsSupportedType( typeof( T ) ) )
         {
             throw new NotSupportedException( $"Requested type {typeof( T ).Name} is not supported" );
         }
```

Comment:
For consistency, and perhaps just for the time being - we might want to call 

```diff
- T newView = new( );
+ (T)ReflectionHelpers.GetDefaultViewInstance( typeof(T))
```

Currently there is differing behaviour between the two Create methods.  e.g. `Button` is not getting its `Text` set with the new method.  Rather than play 'wack-a-mole' by Type we should have it flow through same method.  This may also allow removing the `new()` constraint on `T`.  I will make a longer post on that in the Discussion.

```csharp
    [Test]
    public void TestEquality()
    {
        Assert.That(
            ViewFactory.Create<Button>().Text,
            Is.EqualTo(
                ((Button)ViewFactory.Create(typeof(Button))).Text
                )
            );
    }
```

Reply:
Good catch.

And actually, I have ViewFactory tests in the branch I cherry-picked this from that would have caught that already, as it performs an exhaustive test of all types in the assembly that inherit from View and are assignable to supported types.

I'll add a button case, which will of course catch this. The original never had one and just relied on the activator and the fact that the default case calls the SetActualText method on whatever came back.

When I get to ViewFactoryTests, which I'll move up in my priority list to be the next fixture I hit, I'll be sure all possibilities are covered.

Reply:
As for `new()`, the proposed change doesn't fix that. The activator still implicitly has that constraint, or else you're going to get a runtime error.